test/extended/cli/explain.go

```diff
@@ -427,8 +430,12 @@ func getCrdTypes(oc *exutil.CLI) []schema.GroupVersionResource {
 	crdTypes = append(crdTypes, machineTypes...)
 	crdTypes = append(crdTypes, additionalOperatorTypes...)
 
-	// Only add metal3 types when the baremetal capability is enabled
+	// Conditional, capability-specific types
 	for _, capability := range clusterVersion.Status.Capabilities.EnabledCapabilities {
+		if capability == configv1.ClusterVersionCapabilityMarketplace {
+			crdTypes = append(crdTypes, marketplaceTypes...)
+			break
+		}
```

Comment:
nit: you could get fancy and use a switch statement for extra XP :)

Reply:
> ...use a switch statement...

Done in abf3b8ea6f -> 160ce18f5a.